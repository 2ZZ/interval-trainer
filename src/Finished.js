import { Typography } from "@mui/material";
import * as React from "react";

export default function Finished(props) {
  return (
    <div className="finishedTitle">
      {props.routineFinished !== undefined && props.routineFinished && (
        <>
          <Typography
            component={"span"}
            sx={{
              p: 0,
              display: "flex",
              flexDirection: "column",
              alignItems: "center",
            }}
          >
            <h2>Finished!</h2>
          </Typography>
        </>
      )}
    </div>
  );
}
