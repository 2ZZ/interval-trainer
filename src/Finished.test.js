import React from "react";
import renderer from "react-test-renderer";
import Finished from "./Finished";

it("finished renders as expected", () => {
  const component = renderer.create(<Finished workoutFinished={true} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
