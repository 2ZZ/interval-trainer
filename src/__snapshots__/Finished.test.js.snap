// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`finished renders as expected 1`] = `
<div
  className="finishedTitle"
>
  <style
    dangerouslySetInnerHTML={
      Object {
        "__html": ".css-1kwglvz-MuiTypography-root{margin:0;font-family:\\"Roboto\\",\\"Helvetica\\",\\"Arial\\",sans-serif;font-weight:400;font-size:1rem;line-height:1.5;letter-spacing:0.00938em;padding:0px;display:-webkit-box;display:-webkit-flex;display:-ms-flexbox;display:flex;-webkit-flex-direction:column;-ms-flex-direction:column;flex-direction:column;-webkit-align-items:center;-webkit-box-align:center;-ms-flex-align:center;align-items:center;}",
      }
    }
    data-emotion="css 1kwglvz-MuiTypography-root"
  />
  <span
    className="MuiTypography-root MuiTypography-body1 css-1kwglvz-MuiTypography-root"
  >
    <h2>
      Finished!
    </h2>
  </span>
</div>
`;
